config.json

```diff
@@ -45,7 +45,7 @@
         "control_flow_loops",
         "lists",
         "maps",
-        "regular_expressions"
+        "lists"
```

Comment:
Please fix this, it has control_flow_loops, lists, maps, and lists again. I would remove the second "lists"

Reply:
Recommend running `configlet fmt config.json` to format this file. You may need to run `fetch-configlet` to update your local version of configlet.

Reply:
Oops, I didn't notice the duplicate.

Reply:
@raphalvessa I would suggest the changes are reverted back to the original version:
> "control_flow_loops",
> "regular_expressions",
> "maps",
> "lists"